packages/ui-editable/src/InPlaceEdit/index.js

```diff
@@ -140,9 +159,16 @@ class InPlaceEdit extends Component {
     const isEditMode = !readOnly && mode === 'edit'
 
     return isEditMode ? (
-      <FocusableView as="span" display="block" focused={showFocusRing}>
+      <View
+        as="span"
+        display="block"
+        withFocusOutline={showFocusRing}
+        position="relative"
+        css={this.props.styles.inPlaceEdit}
+        borderRadius="medium"
+      >
```

Comment:
Luckily this was the only place InstUI used this component. I've added its CSS for 100% backwards compatibility.

Reply:
Hm, seems the component looks the same with and without this extra style, but probably it's best to transfer all props, just in case. Good idea :)

Reply:
yea, it might affect some styled text inside this component